src/files/proxy_headers.conf

```diff
@@ -1,4 +0,0 @@
-proxy_set_header  X-Real-IP  $remote_addr;
```

Comment:
Just to confirm my understanding, this is being deleted because they've either been moved to or already existed within `src/files/sites.nginx` which gets copied to `/etc/nginx/conf.d/notifications.conf` so this just tidies up an extra file with no impact on the end state configuration?

Reply:
Correct - if we had multiple sites configured and were setting this several times, I'd be happy for them to sit in a separate file. But it isn't needed here. They now sit in the sites file directly.

Reply:
yes that makes sense 👍